core/merginapi.cpp

```diff
@@ -1079,14 +1079,29 @@ void MerginApi::createProjectFinished()
   }
   else
   {
-    QString serverMsg = extractServerErrorMsg( r->readAll() );
+    QByteArray data = r->readAll();
+    QVariant code = extractServerErrorValue( data, QStringLiteral( "code" ) );
+    QString serverMsg = extractServerErrorMsg( data );
     QString message = QStringLiteral( "FAILED - %1: %2" ).arg( r->errorString(), serverMsg );
-    CoreUtils::log( "create " + projectFullName, message );
+    bool showLimitReachedDialog = code.toString() == QStringLiteral( "ProjectsLimitHit" );
```

Comment:
Can we define these error codes as a new enum type and compare against enum values instead?

Example:
`... code == ErrorCodes::ProjectsLimitHit`